service/gcs/bridge/connection.go

```diff
@@ -5,48 +5,18 @@ import (
 
 	"github.com/pkg/errors"
 
+	"io"
```

Comment:
If we're keeping with the patterns of our other imports, "io" should be at the top of the import list, then a newline, then external dependencies, then another newline, then internal dependencies.